lib/dealership.rb

```diff
@@ -0,0 +1,16 @@
+class Dealership
+  attr_reader :name, :location
+
+  def initialize(name, location)
+    @name = name
+    @location = location
+  end
+
+  def inventory
+    @inventory = []
+  end
```

Comment:
You want to set this during the initialization that way you can update @inventory later otherwise it will only every return `[]`

```
  attr_reader :name, :location, :inventory

  def initialize(name, location)
    @name = name
    @location = location
    @inventory = []
  end
```